go/chat/server.go

```diff
@@ -1188,8 +1151,16 @@ func (h *Server) PostLocal(ctx context.Context, arg chat1.PostLocalArg) (res cha
 		return res, fmt.Errorf("no TLF name specified")
 	}
 
+	// Check for any slash command hits for an execute
+	if handled, err := h.runSlashCommands(ctx, uid, arg.ConversationID, arg.Msg.ClientHeader.TlfName,
+		arg.Msg.MessageBody); handled {
+		h.Debug(ctx, "PostLocal: handled slash command with error: %s", err)
```

Comment:
only log if error?

Reply:
I like to see it all the time, it isnt going to be spamming anything.